radio/sdcard/horus/SCRIPTS/WIZARD/heli/wizard.lua

```diff
@@ -0,0 +1,559 @@
+local function runAilerConfig(event)
+  lcd.clear()
+  if AilerBackground == nil then
+    AilerBackground = Bitmap.open("img/ail.png")
+  end
+  lcd.drawBitmap(AilerBackground, 0, 0)
+  lcd.drawBitmap(ImgPageUp, 0, 95)
+  lcd.drawBitmap(ImgPageDn, 455, 95)
+  fields = AilerFields
+  lcd.drawText(40, 20, "Aileron Channel")
+  lcd.drawFilledRectangle(40, 45, 100, 30, TEXT_BGCOLOR)
+  fields[1][4]=1
+  local result = runFieldsPage(event)
+  return result
+end
+
+local EleFields = {
+  {50, 50, COMBO, 1, 1, { "CH1", "CH2", "CH3", "CH4", "CH5", "CH6", "CH7", "CH8" } },
+}
+
+local EleBackground
+
+local function runEleConfig(event)
+  lcd.clear()
+  if EleBackground == nil then
+    EleBackground = Bitmap.open("img/ele.png")
+  end
+  lcd.drawBitmap(EleBackground, 0, 0)
+  lcd.drawBitmap(ImgPageUp, 0, 95)
+  lcd.drawBitmap(ImgPageDn, 455, 95)
+  fields = EleFields
+  lcd.drawText(40, 20, "Elevator Channel")
+  lcd.drawFilledRectangle(40, 45, 100, 30, TEXT_BGCOLOR)
+  fields[1][4]=1
+  local result = runFieldsPage(event)
+  return result
+end
+
+local RudBackground
+
+local RudFields = {
+  {50, 50, COMBO, 1, 3, { "CH1", "CH2", "CH3", "CH4", "CH5", "CH6", "CH7", "CH8" } },
+}
+local function runRudConfig(event)
+  lcd.clear()
+  if RudBackground == nil then
+    RudBackground = Bitmap.open("img/rudder.png")
+  end
+  lcd.drawBitmap(RudBackground, 0, 0)
+  lcd.drawBitmap(ImgPageUp, 0, 95)
+  lcd.drawBitmap(ImgPageDn, 455, 95)
+  fields = RudFields
+  lcd.drawText(40, 20, "Rudder (Tail) Channel")
+  lcd.drawFilledRectangle(40, 45, 100, 30, TEXT_BGCOLOR)
+  fields[1][4]=1
+  local result = runFieldsPage(event)
+  return result
+end
+
+local lineIndex
+
+local function drawNextChanelLine(text, text2)
+  lcd.drawText(40, lineIndex, text)
+  lcd.drawText(242, lineIndex, ": CH" .. text2 + 1)
+  lineIndex = lineIndex + 20
+end
+
+local function drawNextSwitchLine(text, text2)
+  lcd.drawText(40, lineIndex, text)
+  lcd.drawText(242, lineIndex, ": " ..switches[text2 + 1])
+  lineIndex = lineIndex + 20
+end
+
+local function drawNextTextLine(text, text2)
+  lcd.drawText(40, lineIndex, text)
+  lcd.drawText(242, lineIndex, ": " ..text2)
+  lineIndex = lineIndex + 20
+end
+
+local function switchLine(text)
+  text=SwitchFields[2][5]
+  getFieldInfo(text)
+  swnum=text.id
+end
+
+local ConfigSummaryFields = {
+  {110, 250, COMBO, 1, 0, { "No, I need to change something", "Yes, all is well, create the model !"} },
+}
+
+local ImgSummary
+
+local function runConfigSummary(event)
+  lcd.clear()
+  fields = ConfigSummaryFields
+  lcd.drawBitmap(BackgroundImg, 0, 0)
+  lcd.drawBitmap(ImgPageUp, 0, 95)
+  lineIndex = 10
+  -- Type
+  if TypeFields[1][5]==0 then
+    drawNextTextLine("Type","FBL")
+  else
+    drawNextTextLine("Type","FB")
+    if TypeFields[2][5]==0 then
+      lcd.drawText(290,10,"Swash 120")
+    elseif TypeFields[2][5]==1 then
+      lcd.drawText(290,10,"Swash 120X")
+    elseif TypeFields[2][5]==2 then
+      lcd.drawText(290,10,"Swash 140") else

```

Comment:
your "else" position and indentation needs to be changed (many instances of that)

Reply:
else position and indentation changed